### PR TITLE
Add downwardApi

### PR DESCRIPTION
**Pull request by danduk82**

### CODE REVIEW

**Review by cryptobioz — APPROVED** [review]

File: templates/_helpers.tpl [hunk 1]
```diff
@@ -124,21 +124,24 @@ tolerations:
 {{- end }}
 {{- end }}
 
-{{- define "common.oneEnv" -}}
-{{- if eq ( default "value" .value.type ) "value" }}
+{{- define "common.oneEnv" }}
 - name: {{ .name | quote }}
```
Comment (sbrunner) [inline templates/_helpers.tpl:128]: It looks that this line will be displayed on type=none ...

Comment (sbrunner) [reply templates/_helpers.tpl:128]: Perhaps we should add a test for that ...

File: tests/expected_service_account.yaml [hunk 2]
```diff
@@ -3,38 +3,38 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: relase-long-name-rel-chart-long-name-cha-value-long-service-nam-env
+  name: release-long-name-re-chart-long-name-cha-value-long-service-nam-env
```
Comment (sbrunner) [inline tests/expected_service_account.yaml:6]: You shouldn't have these changes, already made on master, can you try to do a rebase?

**Review by sbrunner — CHANGES_REQUESTED** [review]
Just some comments

**Review by sbrunner — APPROVED** [review]

**Pull request merged**